datasets/conv_ai/conv_ai.py

```diff
@@ -0,0 +1,99 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""ConvAI: A Dataset of Topic-Oriented Human-to-Chatbot Dialogues"""
+
+from __future__ import absolute_import, division, print_function
+
+import json
+
+import datasets
+
+
+_DESCRIPTION = """\
+ConvAI is a dataset of human-to-bot conversations labelled for quality. \
+This data can be used to train a metric for evaluating dialogue systems. \
+Moreover, it can be used in the development of chatbots themselves: it contains the information \
+on the quality of utterances and entire dialogues, that can guide a dialogue system in search of better answers.
+"""
+
```

Comment:
no citation ?

Reply:
couldn't find any citation info, the dataset and paper is hosted on github